addons/t9n/__manifest__.py

```diff
@@ -4,14 +4,17 @@
     "category": "TODO: find the appropriate category",
     "description": "TODO: write a description of the module",
     "depends": ["base", "web"],
-    "data": [
-        "views/t9n_templates.xml"
-    ],
     "application": True,
     "assets": {
         "web.assets_backend": [
             "t9n/static/src/**/*",
         ],
     },
+    "data": [
+        'security/ir.model.access.csv',
+        "views/t9n_templates.xml",
+        "views/t9n_project_views.xml",
+        "views/t9n_menus_views.xml"
```

Comment:
You may want to use a [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) whenever possible. Say I want to add another item right after the last one in the list, I'll have to add a comma to your line, and my changes will shadow yours in the git blame
```suggestion
        'security/ir.model.access.csv',
        "views/t9n_templates.xml",
        "views/t9n_project_views.xml",
        "views/t9n_menus_views.xml",
```